sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class InterestRateSwapExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
```

Comment:
Consider adding a comment or TODO for why this and GetAndValidatePortfolioCashFlows is empty

Reply:
It'll definitely be a TODO as we would need market data. It'll come in the next iteration when doing valuation. For querying, we don't need data. As with the other ones, slowly filling in the gaps.